java/arcs/core/host/AbstractArcHost.kt

```diff
@@ -211,7 +225,7 @@ abstract class AbstractArcHost(
         runningMutex.withLock { runningArcs.clear() }
         clearContextCache()
         pausedArcs.clear()
-        updateArcHostContextScope.cancel()
+        contextSerializationTasks.cancel()
```

Comment:
would we want to attempt to let the channel drain here? Something like...

```
contextSerializationChannel.close()
contextSerializationTasks.join()
```

Reply:
I thought about it, but don't think it's a good idea here at tear-down flow to block for a long time.
A context serialization takes 400-500 ms and if there are multiple of them, we might cause ANR if the tear-down is run at app lifecycle like onDestroy.

Reply:
corrections: currently the pause() function at the shutdown() would drain all serialization tasks before it actually shuts everything down. So the join() is not necessary and we might hit ANR if the serialization tasks really take a lot of time.